### PR TITLE
Replace lorem ipsum by explanation texts

### PR DESCRIPTION
**Pull request by maudetes**
Close etalab/data.gouv.fr#266
Fix etalab/data.gouv.fr#273

Translations will be extracted in another PR to prevent conflicts.

### CODE REVIEW

File: theme/js/components/search/suggest-results.vue [hunk 1]
```diff
@@ -25,9 +25,7 @@ Used by the suggest feature to display typeahead-style results when you type you
         <span v-html="arrow" />
       </a>
       <p class="text-grey-300 m-0">
-        Nullam neque bibendum convallis enim aliquam. Integer egestas accumsan,
-        in varius lectus. Elementum a facilisis nibh pellentesque enim egestas
-        porta.
+        {{ $t('Datasets are collections of data, i.e. structured information, easily readable by a machine.') }}
```
Comment (restuccia) [inline theme/js/components/search/suggest-results.vue:28]: Pas trop self-explanatory. Je suggère "Datasets are a collection of one or more resources, i.e. data files that can be easily read by a machine"

File: theme/js/components/search/suggest-results.vue [hunk 2]
```diff
@@ -64,9 +62,7 @@ Used by the suggest feature to display typeahead-style results when you type you
         <span v-html="arrow" />
       </a>
       <p class="text-grey-300 m-0">
-        Nullam neque bibendum convallis enim aliquam. Integer egestas accumsan,
-        in varius lectus. Elementum a facilisis nibh pellentesque enim egestas
-        porta.
+        {{ $t('Reuses refer to the use of data for an other purpose than the one they were produced for.') }}
```
Comment (restuccia) [inline theme/js/components/search/suggest-results.vue:65]: That is a very good explanation but perhaps not pertinent to the view?

data.gov.ie says "Datasets have been used to build apps, websites and visualizations. They've been featured in articles, and written about in news reports and blog posts. Showcases collect together the best examples of datasets in use, to provide further insight, ideas, and inspiration."

Canada says "One of the benefits of Open Data is the ability to create applications. On this page you will find mobile and web-based applications, created within the Government of Canada, the public, and Canadian Open Data Experience (CODE) winners and participants"

**Review by restuccia — COMMENTED** [review]
Are these the texts that appear in the research view?

File: theme/js/components/search/suggest-results.vue (hunk 2 above)
Comment (maudetes) [reply theme/js/components/search/suggest-results.vue:65]: How about the following ?
> Public datasets are constantly reused in applications, visualizations, articles, etc.. Some of these reuses are referenced here.

Comment (restuccia) [reply theme/js/components/search/suggest-results.vue:65]: Awesome!

**Pull request merged**